packages/nextra/src/compile.ts

```diff
@@ -83,29 +84,36 @@ export async function compileMdx(
     ...(frontMatter.mdxOptions as Record<string, unknown>)
   }
 
+  const {
+    staticImage,
+    flexsearch,
+    readingTime,
+    latex,
+    codeHighlight,
+    defaultShowCopyCode
+  } = loaderOptions
+
   const compiler =
     (useCachedCompiler && cachedCompilerForFormat[format]) ||
     (cachedCompilerForFormat[format] = createProcessor({
       jsx,
       format,
       outputFormat,
       providerImportSource: 'nextra/mdx',
-      // https://github.com/hashicorp/next-mdx-remote/issues/307#issuecomment-1363415249
-      development: false,
```

Comment:
seems like this is no longer needed on 4.2.1 version https://github.com/hashicorp/next-mdx-remote/issues/307#issuecomment-1377131951